### DISCUSSION TITLE
Supernote Half Page View

### DISCUSSION THREAD

**Discussion by Mactamhais** [kudrykv/latex-yearly-planner#38] [Ideas]
First, I love these planners. Great work.

What I've done is adapt one version to my own use case. I've attached my edited version here (it's edited only for the first three months). But since I don't have *any* Latex skills, any changes I've made are completed by way of editing the PDF in Acrobat. It's clumsy, but it works. The easiest changes are done simply by adding images into various pages in the planner. My monthly view has a habit tracker at the bottom (pasted in the place of the two notes columns that you created). This is quick and easy to do in that there are only twelve months. More time consuming is pasting images into daily views (because there are 365 of them!). I do a month or two at a time to spread out the work.

But what I specifically wanted to mention is a change that might be useful for other Supernote users. I start your version that has the months along the left side. That works for me, but when using the Half Page View in Supernote (where one can see the top or bottom of any page - but in landscape mode so it's larger) I can only see some of the months at any given time. For example, from May onwards: if I'm looking at the top half of the daily view and want to switch back to the monthly view, I can't see those months. It's an extra step to get to that. So I redesigned the months along the left side so that they show all twelve months even in the Half Page View (the months appear twice in the full page view). I find this option is a nice feature for the Supernote - and probably only for the Supernote. See attached sample.

Note that doing this manually in Acrobat is a really tedious process - especially updating all the links. So I thought I'd pass this along and show you how I implemented it in case it was something you wanted to incorporate for others. It would be much easier to do in Latex. I wish I had the Latex skills, but I don't believe I have the time to devote to learning that.

[sn_a5x.mos.lined.default.2023_mosx2.pdf](https://github.com/kudrykv/latex-yearly-planner/files/9192629/sn_a5x.mos.lined.default.2023_mosx2.pdf)

**Comment by kudrykv** [comment]
Hi @Mactamhais!
Thank you for your interest in the project!

I'm considering incorporating some kind of habit tracker into the planner.

Regarding the half-page view: do you always use the planner in that mode? That is an adjustment that is doable but won't come soon. I also wonder how many people use the planner in that mode.

**Comment by Mactamhais** [comment]
Thanks. I understand the half page view to be a popular feature. It was
well received when added as a feature. I switch back and forth regularly
when using it - but i especially use it when reading long PDF documents
like academic papers. The landscape view makes it easier to read and
annotate.

KJM

**Reply by kudrykv** [reply]
Do you have A5X or A6X device?

I was thinking that A6X could deserve its own fitted template. I didn't come up with what it should look like yet, though.

**Comment by Mactamhais** [comment]
I have the A5X. But I understood that your templates, for example, could be
used in either the A5X or the A6X. Would a specific template for the A6X be
necessary? Having said that, I would think that the Split Page View would
be more popular for A6X users because of the smaller screen. If so, then a
template for the A6X that had the double-set on month tabs (left side)
would be useful.

Note also that both the A5X and A6X devices can now move the menu to any
side of the device (top, bottom, left and right) and can be hidden/reviewed
with a double-tap. As such it complicates setting aside part of the page
(template creators like yourself have left part of the top blank because
the menu was anchored there - but no longer).

Kevin MacTavish
|<3\/1/\/  ^^@( 7@\/1$|-|
416-848-7350

**Reply by kudrykv** [reply]
> templates could be used in either the A5X or the A6X

That's true. My thinking is about the screen "real estate" one has. So, A6X is smaller in size, and one would need to zoom it in some way. I come from paper notebooks: one can't zoom in there. Staying that, a paper notebook is usually enough and convenient to use even though it doesn't have zoom. My intention here is the same: have a planner that would be convenient to use even without zoom.

> devices can now move the menu to any side of the device

Yup. That's a new feature I need to account for. I'm thinking of having there a configuration to change the layout and select shifted (for the menu bar) or full-screen (yay, the double-tap) minus corner icon.
